install/.colcon_install_layout

```diff
@@ -0,0 +1 @@
+isolated
```

Comment:
@suab321321 Did you run colcon inside the `rclpy` folder? You should probably run it from the root of the workspace.

Also you might want to clean up the `install`, `log`, `build` folders.

```
git rm -rf build install log
```

Then commit the result and push it.

Reply:
@allenh1  Sir my main problem is when 
`
for callback in self._parameters_callback:
                result = callback(parameter_list)
                if(not result.successful):
                    return result
`
this is showing raising TypeError method is not iteratable.I tried to find the error in stackoverflow but cannot understanding why it is failing here in this case

Reply:
What is the type of `self._parameters_callback`? This error seems to imply that `self._parameters_callback` is a method, _not_ an iterable (such as a list of methods).

But if you could clean up the diff (by removing all the extra files), it would be a lot easier for people to help you fix the issues (since there's fewer changes to look at).

Reply:
- Its type is Callable[[List[Parameter],
SetParameterResult]

I have deleted the log build and install folders,and I have no idea how this file changed I am working with node.py

Should I reinitialize my local repo again with the master?

Reply:
@suab321321 I think you cloned the `rclpy` repo inside this repo and added it.

> Callable[[List[Parameter],
SetParameterResult]

This is not iterable. You want a list of callables, not a callable (which is why the interpreter is upset).

Reply:
@allenh1 I m creating a repo and I will be doing new PR from that.